### DISCUSSION TITLE
Raspberry support - docker

### DISCUSSION THREAD

**Discussion by 1mfaasj** [plausible/analytics#4311] [Self-Hosted Support]
Hi I'm trying to run plausible on my raspberry. When I use specific images it works but not with the most recently ones.
Is there a reason why there is no support for raspberry/arm architecture anymore?

for instance this image works: clickhouse/clickhouse-server:22.6-alpine
but not the recently ones like this image: clickhouse/clickhouse-server:23.3.7.5-alpine

this version also doesn't work with the latest plausible image: ghcr.io/plausible/community-edition:v2.1.1
so I needed to go back to plausible/analytics:v2

it looks like I'm stuck with older versions?
Kind regards

**Comment by ruslandoga** [comment]
👋 @1mfaasj 

ClickHouse forums might be a better place to ask this.

**Reply by ruslandoga** [reply]
My guess would be that ClickHouse tries to be fast and keeps using more specialized instructions which might not always be supported on Raspberry Pis.

**Reply by ruslandoga** [reply]
Seems like your question has been answered in https://github.com/ClickHouse/ClickHouse/issues/66021#issuecomment-2206007916
